generators/xml2jsonFile.ts

```diff
@@ -0,0 +1,72 @@
+import * as fs from "fs";
+import * as path from "path";
+import * as xml2js from "xml2js";
+
+type Config = {
+  icd10: {
+    xmlPath: string;
+  };
+};
+
+const parentPath = process.argv[2] || path.resolve(process.cwd(), "..", "..");
+
+(async () => {
+  const packageJson = await readPackageJson();
+  const xmlPath = readXmlPathFromParentPackageJson(packageJson);
+  const icd10xml = await readXml(xmlPath);
+  const classificationJson = await parseXml(icd10xml.toString());
+  await saveIcdJson(classificationJson);
+  console.log("Json generated successfully!");
```

Comment:
Is it intentional? If so, maybe use `info` instead?

Reply:
Yes it is intentional. From my side it could be a `info`. Could you tell me please why would you prefer `info` instead of `log`?